src/VfacTmdb/Results/ItemChange.php

```diff
@@ -0,0 +1,198 @@
+<?php declare(strict_types = 1);
+/**
+ * This file is part of the Tmdb package.
+ *
+ * (c) Vincent Faliès <vincent@vfac.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Vincent Faliès <vincent@vfac.fr>
+ * @copyright Copyright (c) 2017-2021
+ */
+
+namespace VfacTmdb\Results;
+
+use VfacTmdb\Abstracts\Results;
+use VfacTmdb\Interfaces\TmdbInterface;
+use VfacTmdb\Interfaces\Results\ItemChangeResultsInterface;
+
+/**
+ * Class to manipulate an ItemChange result
+ * @package Tmdb
+ * @author Steve Richter <steve@nerdbra.in>
+ * @copyright Copyright (c) 2021
+ */
+class ItemChange extends Results implements ItemChangeResultsInterface
+{
+    /**
+     * Id
+     * @var string
+     */
+    protected $id;
+    /**
+     * Key
+     * @var string
+     */
+    protected $key;
+    /**
+     * Action
+     * @var string
+     */
+    protected $action;
+    /**
+     * Time
+     * @var \DateTime
+     */
+    protected $time;
+    /**
+     * iso_639_1
+     * @var string
+     */
+    protected $iso_639_1 = null;
+    /**
+     * iso_3166_1
+     * @var string
+     */
+    protected $iso_3166_1 = null;
+    /**
+     * Value
+     * @var array
+     */
+    protected $value;
+    /**
+     * Original value
+     * @var array
+     */
+    protected $original_value = null;
+
+    /**
+     * Constructor
+     * @param TmdbInterface $tmdb
+     * @param \stdClass $result
+     */
+    public function __construct(TmdbInterface $tmdb, \stdClass $result)
+    {
+        $result = $this->initResultObject($result);
+
+        parent::__construct($tmdb, $result);
+
+        // Populate data
+        $this->id             = $this->data->id;
+        $this->key            = $this->data->key;
+        $this->action         = $this->data->action;
+        $this->time           = $this->data->time;
+        $this->iso_639_1      = $this->data->iso_639_1;
+        $this->iso_3166_1     = $this->data->iso_3166_1;
+        $this->value          = $this->data->value;
+        $this->original_value = $this->data->original_value;
+    }
+
+    /**
+     * initResultObject
+     * @param  \stdClass  $result
+     * @return \stdClass
+     */
+    private function initResultObject(\stdClass $result) : \stdClass
```

Comment:
This function attempts to normalize the API data as the API is very inconsistent with the results here:

- Sometimes it does not include the `original_value` key.
- Sometimes it does not include the `value` key.
- Sometimes the `value` key is an int/string, other times it is an array.